ui/shared/components/panel/variants/VariantGene.jsx

```diff
@@ -322,7 +322,7 @@ const INDIVIDUAL_NAME_COLUMN = { name: 'individualName', content: '', format: ({
 const RNA_SEQ_COLUMNS = [
   INDIVIDUAL_NAME_COLUMN,
   ...RNA_SEQ_DETAIL_FIELDS.map(name => (
-    { name, content: camelcaseToTitlecase(name).replace(' ', '-'), format: row => row[name].toPrecision(3) }
+    { name, content: camelcaseToTitlecase(name), format: row => row[name].toPrecision(3) }
```

Comment:
RNA behavior needs to stay the same, this should not change. I requested this change for the phenotype hover onl